src/js/modules/sections/components/SectionPage.js

```diff
@@ -3,26 +3,121 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import injectSheet from 'react-jss';
-import sections from '../../sections';
+import { Grid, Row, Col } from 'react-bootstrap/lib/';
 
 import articles from '../../articles';
 
 const styles = {
   SectionPage: {
     color: '#000',
+  },
+  sectionName: {
+    width: '1080px',
+    height: '43px',
```

Comment:
The width/height of the textbox matter is also why, as seen in the screenshot below, if there are no subsections for a top level section, there is still a weird empty space between the section title and the line that separates it from the article list.
--
![image](https://user-images.githubusercontent.com/15334952/28792427-8e59d874-75fd-11e7-93b1-a8a061aa9524.png)